CONTRIBUTING.md

```diff
@@ -1,46 +1,90 @@
 # How to contribute
 
-etcd is Apache 2.0 licensed and accepts contributions via GitHub pull requests. This document outlines some of the conventions on commit message formatting, contact points for developers, and other resources to help get contributions into etcd.
+etcd is Apache 2.0 licensed and accepts contributions via GitHub pull requests.
+This document outlines basics of contributing to etcd.
 
-# Email and chat
+This is a rough outline of what a contributor's workflow looks like:
+* Find something to work on
+* Setup development environment
+* Implement your change
+* Commit your change
+* Create a pull request
+* Get your pull request reviewed
 
-- Email: [etcd-dev](https://groups.google.com/forum/?hl=en#!forum/etcd-dev)
-- Slack: [#etcd](https://kubernetes.slack.com/messages/C3HD8ARJ5/details/)
+If you have any questions about, please reach out using one of the methods listed in [contact].
 
-## Getting started
+[contact]: ./README.md#Contact
 
-- Fork the repository on GitHub
-- Read the README.md for build instructions
+## Learn more about etcd
 
-## Reporting bugs and creating issues
+Before making a large change please look through resources below learn more about etcd and tools used for development.
 
-Reporting bugs is one of the best ways to contribute. However, a good bug report has some very specific qualities, so please read over our short document on [reporting bugs](https://etcd.io/docs/latest/reporting_bugs) before submitting a bug report. This document might contain links to known issues, another good reason to take a look there before reporting a bug.
+* Please learn about [Git](https://github.com/git-guides) version control system used in etcd.
+* Watch [etcd deep dive](https://www.youtube.com/watch?v=D2pm6ufIt98&t=927s)
+* Watch [etcd code walk through](https://www.youtube.com/watch?v=H3XaSF6wF7w)
```

Comment:
I think we should also add etcd.io reference. e.g. Learn more at etcd.io

Reply:
And also good to mention that refer to the dev docs provided here https://github.com/etcd-io/etcd/tree/main/Documentation/contributor-guide